content/en/docs/concepts/storage/volumes.md

```diff
@@ -462,92 +462,155 @@ and then removed entirely in the v1.26 release.
 
 ### hostPath {#hostpath}
 
-{{< warning >}}
-HostPath volumes present many security risks, and it is a best practice to avoid the use of
-HostPaths when possible. When a HostPath volume must be used, it should be scoped to only the
-required file or directory, and mounted as ReadOnly.
-
-If restricting HostPath access to specific directories through AdmissionPolicy, `volumeMounts` MUST
-be required to use `readOnly` mounts for the policy to be effective.
-{{< /warning >}}
-
 A `hostPath` volume mounts a file or directory from the host node's filesystem
 into your Pod. This is not something that most Pods will need, but it offers a
 powerful escape hatch for some applications.
 
+{{< warning >}}
```

Comment:
One more warning potentially worth adding, based on a few production bugs I've encountered:

Hostpath on "/" is strongly discouraged. Not only does it present security risks, but it can interfere with Kubernetes volume unmounting when used with the default "None" [mount propagation](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) . Use "HostToContainer" mount propagation if you must.

Reply:
Can we save that for a different PR? This PR has a different aim.

Reply:
I tend to agree with @msau42 that adding that note (warning) would be desirable.

Reply:
I agree! Can we save that change for a different PR? If not: why not?